### PR TITLE
User administration to utilise backend

### PR DESCRIPTION
**Pull request by Jade-Codes**
### What does this PR do?
Updates the admin users page to utilise the backend instead of user story.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: api/requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ blinker==1.6.2
 cachetools==5.3.1
 certifi==2023.7.22
 click==8.1.3
-cryptography==41.0.3
+cryptography
```
Comment (Nepomuceno) [inline api/requirements.txt:8]: Why Rolling back in this ?

File: client/src/api/users/queries.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import { UserDefinition } from '../Models';
+import { useAccessToken } from '@/hooks/use-access-token';
+
+export function useGetUsers() {
+  const accessToken = useAccessToken();
+  return useQuery<UserDefinition[]>({
```
Comment (Nepomuceno) [inline client/src/api/users/queries.tsx:8]: Nice use of the new use query Signature ;)

File: client/src/hooks/use-access-token.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        setAccessToken(accessTokenResponse?.accessToken);
+      })
+      .catch((error) => {
+        if (error instanceof InteractionRequiredAuthError) {
+          instance.acquireTokenPopup(accessTokenRequest).then(function (accessTokenResponse) {
+            setAccessToken(accessTokenResponse?.accessToken);
+          });
+        }
+      });
+  }, [instance]);
```
Comment (Nepomuceno) [inline client/src/hooks/use-access-token.tsx:30]: this should also have a dependency on config.data?.appId and appAuthority.

File: client/src/pages/admin/pages/users.tsx [hunk 4]
```diff
@@ -1,88 +1,40 @@
...
-                  {user?.memberOf?.map((group) => (
-                    <p>{group.displayName}</p>
-                  ))}
-                </td>
-              </>
-            </tr>
-          ))}{' '}
-        </tbody>
-      </table>
+      {data && data?.length > 0 ? (
```
Comment (Nepomuceno) [inline client/src/pages/admin/pages/users.tsx:10]:
```suggestion
      {data && data?.length > 0 && (
```

**Review by Nepomuceno — COMMENTED** [review]
Looking really nice just some small comments.

File: api/requirements.txt (hunk 1 above)
Comment (Jade-Codes) [reply api/requirements.txt:8]: When adding msal it complains of conflicting dependency

**Review by Nepomuceno — APPROVED** [review]
sealOfApproval-1

**Pull request merged**